pxf/pxf-service/src/main/resources/pxf-profiles-default.xml

```diff
@@ -46,6 +46,7 @@ under the License.
         <description>
             This profile is suitable for using when connecting to Hive.
             Supports GPDBWritable output format, as specified in FORMAT header parameter.
+            Based on storage format, it uses one of optimized profiles(HiveRC, HiveText, HiveORC) whenever it's possible.
         </description>
```

Comment:
Does it also need to be changed in Ambari since there is a corresponding template there ?